projects/smoldot-extension/src/utils/types.tsx

```diff
@@ -1 +1,20 @@
-export type NetworkTypes = 'kusama' | 'polkadot' | 'westend' | 'kulupu'
\ No newline at end of file
+export type NetworkTypes = 'kusama' | 'polkadot' | 'westend' | 'kulupu'
```

Comment:
Can we move the types into the root please?  `./src/types.ts`  (also not `tsx`)  that's the same pattern as in polkadotjs  - local types in a `types.ts` file for the folder ... shared types in the shared context usually `../types.ts`